network/nxos/nxos_config.py

```diff
@@ -93,7 +93,7 @@
         without first checking if already configured.
     required: false
     default: false
-    choices: BOOLEANS
+    choices: [ "yes", "no" ]
```

Comment:
Actually, I have been pushing True/False .... we really don't have a set policy on this.